### PR TITLE
Full integration with emoji-mart and phasing out `@bangle.dev/emoji`

### PR DESCRIPTION
**Pull request by Devorein**
Integrated `emoji-mart` package in the following places

1. Page icon
2. Sidebar page navigation icon
3. Callout icon
4. `:` to trigger `emoji-mart`

Please check out these four areas while reviewing

### CODE REVIEW

**Comment by motechFR** [comment]
Looks good!

**Review by motechFR — APPROVED** [review]

File: components/common/NewPageMenu.tsx [hunk 1]
```diff
@@ -69,11 +74,11 @@ export default function NewPageMenu ({ addPage, tooltip, ...props }: Props) {
         onClose={handleClose}
       >
         <MenuItem onClick={() => createPage({ type: 'page' })}>
-          <ListItemIcon><StyledArticleIcon fontSize='small' /></ListItemIcon>
+          <ListItemIcon><StyledArticleIcon fontSize='small' isLightMode={theme.palette.mode === 'light'} /></ListItemIcon>
           <Typography sx={{ fontSize: 15, fontWeight: 600 }}>Add Page</Typography>
         </MenuItem>
         <MenuItem onClick={() => createPage({ type: 'board' })}>
-          <ListItemIcon><StyledDatabaseIcon fontSize='small' /></ListItemIcon>
+          <ListItemIcon><StyledDatabaseIcon fontSize='small' isLightMode={theme.palette.mode === 'light'} /></ListItemIcon>
```
Comment (mattcasey) [inline components/common/NewPageMenu.tsx:81]: we should be able to do this with simple CSS and 'color' property, and 'fill' set to 'currentColor'

**Review by mattcasey — CHANGES_REQUESTED** [review]
Did you try applying colors to icons using CSS first?

File: components/common/NewPageMenu.tsx (hunk 1 above)
Comment (mattcasey) [reply components/common/NewPageMenu.tsx:81]: Actually i'm not sure what this is fixing. Icons look fine to me in production, tested in Chrome and Firefox:
image
image

**Comment by Devorein** [comment]
fill="currentColor" would not turn down the opacity in dark mode, it seemed like in light mode the icons were a bit transparent mainly due to the background being white. But I think it's fine to leave it as it is. I've reverted the style changes back to their original state

**Pull request merged**